app/buck2_client/src/commands/test.rs

```diff
@@ -157,6 +157,36 @@ If include patterns are present, regardless of whether exclude patterns are pres
     #[clap(long)]
     test_executor_stderr: Option<OutputDestinationArg>,
 
+    #[clap(
+        long,
+        group = "default-info",
+        help = "Also build default info (this is not the default)"
+    )]
+    build_default_info: bool,
+
+    #[allow(unused)]
+    #[clap(
+        long,
+        group = "default-info",
+        help = "Do not build default info (this is the default)"
+    )]
+    skip_default_info: bool,
+
+    #[clap(
+        long,
+        group = "run-info",
+        help = "Also build runtime dependencies (this is not the default)"
+    )]
+    build_run_info: bool,
+
+    #[allow(unused)]
+    #[clap(
+        long,
+        group = "run-info",
+        help = "Do not build runtime dependencies (this is the default)"
+    )]
+    skip_run_info: bool,
```

Comment:
do we need this? seems unused,  not passing build_run_info should keep the default?

Reply:
This is just keeping symmetry as @JakobDegen asked earlier in this PR. My understanding is that the point is to be able to explicitly opt-in/out in case defaults change.

I don't actually have a strong opinion on UX here btw. In fact, I would personally prefer not needing those flags and having `buck2 test` also build the targets that match the patterns (at least by default), but apparently some people at Meta pushed back against this :shrug: